pkg/storagecapabilities/storagecapabilities.go

```diff
@@ -105,6 +105,8 @@ var CapabilitiesByProvisionerKey = map[string][]StorageCapabilities{
 	"manila.csi.openstack.org": {{rwx, file}},
 	// ovirt csi
 	"csi.ovirt.org": createRWOBlockAndFilesystemCapabilities(),
+	// huawei
+	"csi.huawei.com": {{rwx, block}},
```

Comment:
Does it only support rwx, block or does it also support rwo, block and rwo, file? If so can you add it like this?
```go
"csi.huawei.com": {{rwx, block}, {rwo, block}, {rwo, file}},
```

Reply:
As far as I understand from [CNV-41531](https://issues.redhat.com/browse/CNV-41531) comments, they only have storage profile configured for iscsi storage class `rwx, block`

Reply:
Maybe I am not seeing it, but I don't see any specifics on what that storage in the JIRA card. But I just checked the user guide for the csi driver and it says this
```
RWO/ROX/RWOP: supported by all types of volumes. RWOP is supported only by kubernetes 1.22 and later versions.
RWX: supported only by Raw Block volumes and NFS volumes
```
So it looks like it should be what I suggested.

Reply:
So I added `{rwx, block}, {rwo, block}, {rwo, file}, {rox, block}, {rox, file}` as it says